controls/srg_ctr/SRG-APP-000100-CTR-000200.yml

```diff
@@ -6,4 +6,8 @@ controls:
     within {{{ full_name }}}.
   related_rules:
   - audit_profile_set
-  status: inherently met
+  rules:
+  - auditd_log_format
+  - auditd_data_disk_error_action
+  - auditd_data_retention_max_log_file_action_stig
```

Comment:
Hmm I'm not sure what the issue is?
So in general the majority of the rules in those SRGs (including the ones you listed) are referring to row 21 in the spreadsheet (search for CCI-000134 there is only one row). That one is covered by these rules I think:
```
  - auditd_log_format
  - auditd_data_disk_error_action
  - auditd_data_retention_max_log_file_action_stig
```
the remediation for them is IIRC the same, setting the proper auditd.conf, just the individual checks look at different properties of auditd.conf. Then some SRGs, namely SRG-APP-000109-CTR-000215 and  SRG-APP-000357-CTR-000800 have an extra rule, 215 is additionally making sure that an alert for audit exists (CCI-000140) and 800 is making sure that partitioning is set up (CCI-001849). This will be...challenging to express in the generated spreadsheet, yes, but as far as the rules go, I don't see the issue.

Can you elaborate?

Reply:
Sorry, I should have been more verbose.
I was checking the SRGs in `shared/references/disa-ctr-srg-v1r3.xml` and  `175`, `180`, `185` `190`, `195` and `200` are about contents of the audit logs, like type, time,  source of events, associated users, containers....

So the rules about disk error and max log file action don't seem to fit under these SRGs.

Reply:
Right, so there is two bits of STIG development history to be aware of.

One is that initially we used to have a separate RHCOS STIG based on the generic OS STIG, but that was scrapped and instead we tried to look at the OS controls we selected and re-map them onto the container platform STIG. This is to say that the mapping is probably not 1:1.

The other reason is that while we were reviewing the STIG with Mantech, they tried to collapse SRGs to avoid having checks and fixes that are almost the same, maybe just checking for the same attribute in different contexts or setting the same fix but checking for two different attributes. I can link you to the historical STIG drafts and you'd note that the historical drafts had 200+ rows, the final draft I'm working off has just 106. This is the result of de-duping and collapsing the SRGs.

In general looking at the file ` shared/references/disa-ctr-srg-v1r3.xml` is a good idea, but I would suggest to base off reviews on the STIG draft even more. There you would see that there is a single row for `SRG-APP-000099-CTR-000190, SRG-APP-000097-CTR-000180, SRG-APP-000098-CTR-000185, SRG-APP-000100-CTR-000195, SRG-APP-000100-CTR-000200, SRG-APP-000109-CTR-000215, SRG-APP-000290-CTR-000670, SRG-APP-000357-CTR-000800` that sets the common auditd.conf and checks for parameters like enriched log format and actions on disk error or disk full. Additionally, two of the SRGs (215 and 670 IIRC) add their own "extra" rules.

Reply:
@jhrozek Thanks, this clarifies a lot.
It was not clear to me that some rows had been merged to make it leaner.